setup.py

```diff
@@ -54,7 +54,7 @@
     "reana-db>=0.8.0a1,<0.9.0",
     "htcondor==8.9.7",
     "retrying>=1.3.3",
-    "paramiko[gssapi]>=2.6.0",
+    "paramiko>=2.7.1",
```

Comment:
We probably need to update `requirements.txt` via [`pip-compile`](https://github.com/jazzband/pip-tools) if we go for this change.

Reply:
Yeah, if it's fine to just keep `paramiko`, I will make changes here as well. 👍